bloom/commands/release.py

```diff
@@ -679,10 +679,31 @@ def open_pull_request(track, repository, distro, interactive, override_release_r
             # Check if a fork already exists on the user's account
 
             try:
-                repo_forks = gh.list_forks(base_info['org'], base_info['repo'])
-                user_forks = [r for r in repo_forks if r.get('owner', {}).get('login', '') == gh.username]
-                # github allows only 1 fork per org as far as I know. We just take the first one.
-                head_repo = user_forks[0] if user_forks else None
+                # There are a lot of forks of the ros/rosdistro repository so
+                # listing those forks takes a very long time.
+                # Let's try a little shortcut by checking if the repository of
+                # the same name owned by the current user, if that repository
```

Comment:
You may want to make this comment a little clearer on what you mean by the "current user".  That is, it's the Github current user, not the Unix current user or anything like that.

```suggestion
                # the same name owned by the current Github user, if that repository
```

Reply:
Done in a790f09. Although I spelled `GitHub` with the preferred capitalization because I'm deeply programmed to do so.

Reply:
> Although I spelled `GitHub` with the preferred capitalization because I'm deeply programmed to do so.

That's fine, I made Scott change "RedHat" to "Red Hat" in the RHEL documentation for the same reasons :).